typesafety/tests/test_autodecorator.py

```diff
@@ -76,3 +76,7 @@ def __unload_test_module(self, name):
     def test_immutable_class_attributes_are_not_decorated(self):
         self.assertEquals(1, self._module.ClassWithSlots().immutable)
         self.assertEquals(1234, self._module.ClassWithSlots().mutable)
+
+    def test_dont_decorate_objects_not_native_to_the_module(self):
```

Comment:
Minor note: if you have typesafety installed and you just run typesafety, the imported one will be the unchanged one. Is this still a problem when you run the tests with tox?

Nitpicking: same difference, IMO